src/main/java/stream/flarebot/flarebot/util/general/FormatUtils.java

```diff
@@ -0,0 +1,172 @@
+package stream.flarebot.flarebot.util.general;
+
+import net.dv8tion.jda.core.entities.Guild;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.Period;
+import org.joda.time.format.PeriodFormatter;
+import org.joda.time.format.PeriodFormatterBuilder;
+import stream.flarebot.flarebot.objects.GuildWrapper;
+
+import java.awt.*;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.concurrent.TimeUnit;
+
+public class FormatUtils {
+    private static DateTimeFormatter timeFormat = DateTimeFormatter.ofPattern("MMMM yyyy HH:mm:ss");
+
+    private static final SimpleDateFormat preciseFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SS");
+
+    private static final PeriodFormatter prettyTime = new PeriodFormatterBuilder()
+            .appendDays().appendSuffix(" Day ", " Days ")
+            .appendHours().appendSuffix(" Hour ", " Hours ")
+            .appendMinutes().appendSuffix(" Minute ", " Minutes ")
+            .appendSeconds().appendSuffix(" Second", " Seconds")
+            .toFormatter();
+
+    /**
+     * Formats a duration.
```

Comment:
Put more description here, how does it format it. What will it look like, what will the param do etc. Having just the @param isn't enough